src/renovate.ts

```diff
@@ -53,7 +53,18 @@ class Renovate {
       dockerArguments.push(`--user ${dockerUser}`);
     }
 
-    dockerArguments.push('--volume /tmp:/tmp', '--rm', this.docker.image());
+    const renovateBaseDirEnvVar = this.input
```

Comment:
do not run the get ENV bars twice, refactor the other call to reuse the value here